pages/index.tsx

```diff
@@ -192,10 +193,22 @@ const Tools: React.FC<{ variant: string }> = ({ variant }) => {
 
 const Home: React.FC = () => {
     const [showAlert, setShowAlert] = useState(true);
+    const [user, setUser] = useState<any | null>(null);
+
     const variant = useMediaQuery({ query: `(max-width: 925px)` })
         ? "mobile"
         : "desktop";
 
+    useEffect(() => {
+        Auth.currentAuthenticatedUser()
```

Comment:
You can wrap the asynchronous code inside a use effect in an async function to use async await syntax